flatgeobuf/hilbert-r-tree.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: FlatGeobuf Spatial Index
+subtitle: Breaking down the Hilbert Packed R-Tree
+---
+
+FlatGeobuf supports optionally including a spatial index that enables random access for each geometry in the file.
+
+## When to use
+
+When _writing_ a FlatGeobuf file, one must decide whether to include a spatial index. A spatial index cannot be added to a FlatGeobuf file after the file has been written.
+
+A spatial index can enable much more efficient reading from FlatGeobuf, by allowing the reader to skip over portions of the file that fall outside of a qiven spatial query region.
+
+## Technical details
+
+In this section we'll get into some gory technical details of how FlatGeobuf's spatial index works. Understanding the below isn't necessary for using FlatGeobuf, but can provide some extra context.
```

Comment:
```suggestion
In this section we'll get into some gory technical details of how FlatGeobuf's spatial index works. Understanding the below isn't necessary for using FlatGeobuf, but it may help to understand how to create and work with FlatGeobuf files.
```